presto-product-tests/conf/docker/common/standard.yml

```diff
@@ -12,7 +12,7 @@ services:
     image: ${HADOOP_MASTER_IMAGE}
     hostname: hadoop-master
     ports:
-      - '1080:1080'
+      - '${HIVE_PROXY_PORT}:1080'
```

Comment:
I think it's right. I checked locally before pushing. There's difference between docker-compose and docker syntax.